### PR TITLE
Jetbrains passthroughs.

### PR DESCRIPTION
**Pull request by anthonyshew**

### CODE REVIEW

File: crates/turborepo-lib/src/task_hash.rs [hunk 1]
```diff
@@ -477,6 +477,9 @@ impl<'a> TaskHasher<'a> {
                         "COLORTERM",
                         "TERM",
                         "TERM_PROGRAM",
+                        // Jetbrains IDE
+                        "JB_IDE_HOST",
+                        "JB_IDE_PORT",
```
Comment (chris-olszewski) [inline crates/turborepo-lib/src/task_hash.rs:482]: We might need more based on [this comment](https://github.com/containers/toolbox/issues/618#issuecomment-1858311856)
```suggestion
                        // Jetbrains IDE
                        "JB_IDE_*",
                        "JB_INTERPRETER",
```

**Review by chris-olszewski — APPROVED** [review]

File: crates/turborepo-lib/src/task_hash.rs [hunk 2]
```diff
@@ -477,6 +477,9 @@ impl<'a> TaskHasher<'a> {
                         "COLORTERM",
                         "TERM",
                         "TERM_PROGRAM",
+                        // Jetbrains IDE
+                        "JB_IDE_*",
+                        "JB_INTERPRETER",
```
Comment (anthonyshew) [inline crates/turborepo-lib/src/task_hash.rs:482]:
```suggestion
                        "JB_INTERPRETER",
                        "_JETBRAINS_TEST_RUNNER_RUN_SCOPE_TYPE"
```

File: crates/turborepo-lib/src/task_hash.rs [hunk 3]
```diff
@@ -477,6 +477,10 @@ impl<'a> TaskHasher<'a> {
...
+                        "_JETBRAINS_TEST_RUNNER_RUN_SCOPE_TYPE"
```
Comment (anthonyshew) [inline crates/turborepo-lib/src/task_hash.rs:483]:
```suggestion
                        "_JETBRAINS_TEST_RUNNER_RUN_SCOPE_TYPE",
```

**Comment by anthonyshew** [comment]
Eagle eye, good find.

**Pull request merged**